nuitka/nodes/BuiltinAnyNodes.py

```diff
@@ -0,0 +1,56 @@
+#     Copyright 2019, Kay Hayen, mailto:kay.hayen@gmail.com
+#
+#     Part of "Nuitka", an optimizing Python compiler that is compatible and
+#     integrates with CPython, but also works on its own.
+#
+#     Licensed under the Apache License, Version 2.0 (the "License");
+#     you may not use this file except in compliance with the License.
+#     You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#     Unless required by applicable law or agreed to in writing, software
+#     distributed under the License is distributed on an "AS IS" BASIS,
+#     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#     See the License for the specific language governing permissions and
+#     limitations under the License.
+#
+""" Node for the calls to the 'any' built-in.
+
+"""
+
+from nuitka.specs import BuiltinParameterSpecs
+
+from .ExpressionBases import ExpressionBuiltinSingleArgBase
+from .shapes.BuiltinTypeShapes import ShapeTypeBool
+
+
+class ExpressionBuiltinAny(ExpressionBuiltinSingleArgBase):
+    kind = "EXPRESSION_BUILTIN_ANY"
+
+    builtin_spec = BuiltinParameterSpecs.builtin_any_spec
+
+    def getIntegerValue(self):
```

Comment:
This function `getIntegerValue` is tasked to say what the node would be should it be considered as an integer. For `any` it ought to be removed.

I am nearly finished on pushing API documentation to the public. The base class has a documentation for the methods to overload.

Reply:
Thank you so much. API will be really helpful.